actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -84,7 +84,7 @@ def format(formatter, filter = nil)
 
       def filter_routes(filter)
         if filter
-          @routes.select { |route| route.defaults[:controller] == filter }
+          @routes.select { |route| route.defaults[:controller] == filter.underscore.sub(/_controller$/, '') }
```

Comment:
Should we move the underscore and sub outside of the loop? That might slow down apps with many routes.